homeassistant/components/igloohome/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+  "domain": "igloohome",
+  "name": "igloohome",
+  "codeowners": ["@keithle888"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/igloohome",
+  "iot_class": "cloud_polling",
+  "requirements": ["igloohome-api==0.0.5"]
```

Comment:
we require proper releases on your repo (with changelog and tag), so please add one there

Reply:
Done, tags are already present. Added a release. I've also added a github action that creates releases automatically for future updates.